egs/wsj/s5/steps/dur_model/rescore.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2015 Hossein Hadian
+
+duration_model_scale=0.5
```

Comment:
describe each variable, add a comment on the top describing what this script does.